save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/projectcollection/CollectionView.kt

```diff
@@ -104,12 +103,11 @@ class CollectionView : AbstractView<CreationViewProps, State>() {
                 projectsTable {
                     getData = { _, _ ->
                         val response = post(
-                            url = "$apiUrl/projects/not-deleted",
-                            headers = Headers().also {
-                                it.set("Accept", "application/json")
-                            },
-                            body = undefined,
+                            url = "$apiUrl/projects/by-filters",
+                            headers = jsonHeaders,
+                            body = Json.encodeToString(ProjectFilters.empty),
```

Comment:
suggestion to specify explicitly ProjectFilters to show that we search CREATED projects:
```suggestion
                            body = Json.encodeToString(ProjectFilters("", listOf(CREATED))),
```

or create a new const with proper name

Reply:
Probably we should just rename `ProjectFilters.empty` -> `ProjectFilters.created` then?

Reply:
Done